spring-boot-ecommerce/src/main/java/com/luv2code/ecommerce/entity/Product.java

```diff
@@ -50,4 +50,84 @@ public class Product {
     @Column(name = "last_updated")
     @UpdateTimestamp
```

Comment:
Why you need this?

Reply:
This will be needed in database point of view.